arangod/RestHandler/RestDocumentHandler.cpp

```diff
@@ -45,7 +45,12 @@ using namespace arangodb::rest;
 
 RestDocumentHandler::RestDocumentHandler(application_features::ApplicationServer& server,
                                          GeneralRequest* request, GeneralResponse* response)
-    : RestVocbaseBaseHandler(server, request, response) {}
+    : RestVocbaseBaseHandler(server, request, response) {
+  if(request->requestType() == rest::RequestType::POST
+      && request->contentLength() > 120) {
```

Comment:
```suggestion
      && request->contentLength() <= 120) {
```
Otherwise only huge documents will be direct, exactly not what we want